java/lib/build.gradle.kts

```diff
@@ -12,12 +12,12 @@ plugins {
 }
 
 group = "io.amplica.graphsdk"
-val uploadedBinariesVersion = "0.0.1-rc1"
+val uploadedBinariesVersion = "1.0.0"
 java.sourceCompatibility = JavaVersion.VERSION_17
 version = if (project.hasProperty("projVersion")) {
     project.properties["projVersion"]!!
 } else {
-    "0.0.2-SNAPSHOT"
+    "1.0.1-SNAPSHOT"
```

Comment:
since snapshot should be higher than the last released on incrementing the patch to make sure it would show up higher but it wouldn't matter from publishing packages perspective since we are not publishing any snapshots.